src/pomerol/TwoParticleGFPart.cpp

```diff
@@ -40,18 +38,37 @@ TwoParticleGFPart::NonResonantTerm& TwoParticleGFPart::NonResonantTerm::operator
     return *this;
 }
 
+MPI_Datatype TwoParticleGFPart::NonResonantTerm::mpi_datatype() {
```

Comment:
Igor, mind writing a docstring on what this is? I realize it's a replacement to serialization.